packages/kmod-6.1-nvidia-r570/kmod-6.1-nvidia-r570.spec

```diff
@@ -331,6 +331,23 @@ install kernel-open/nvidia-peermem.ko %{buildroot}%{_cross_datadir}/nvidia/open-
 install kernel-open/nvidia-drm.ko %{buildroot}%{_cross_datadir}/nvidia/open-gpu/drivers/
 # end open driver
 
+install -d %{buildroot}%{_cross_datadir}/vulkan/icd.d
+install -m 0644 nvidia_icd.json %{buildroot}%{_cross_datadir}/vulkan/icd.d/nvidia_icd.json
+install -d %{buildroot}%{_cross_datadir}/vulkan/implicit_layer.d/
+install -m 0644 nvidia_layers.json %{buildroot}%{_cross_datadir}/vulkan/implicit_layer.d/nvidia_layers.json
+# install -m 0644 %{S:208} %{buildroot}%{_cross_datadir}/vulkan/icd.d/nvidia_icd.%{_cross_arch}.json
+# ln -s nvidia_icd.%{_cross_arch}.json \
+#       %{buildroot}%{_cross_datadir}/vulkan/icd.d/nvidia_icd64.json
```

Comment:
To be consistent with the [`nvidia-container-runtime`] you can install `nvidia_layers.json` at `%{buildroot}%{_cross_datadir}/vulkan/icd.d`, and symlink to that from `%{buildroot}%{_cross_datadir}/vulkan/implicit_layer.d/`

[`nvidia-container-runtime`]: https://github.com/NVIDIA/nvidia-container-toolkit/blob/132c9afb6cfa1e6ae0a8f5170566de4b677d2606/internal/discover/graphics.go#L88

Reply:
Good catch, will do!